.travis.yml

```diff
@@ -59,7 +59,14 @@ install:
     - sudo apt-get install cmake
 
     # Install Mongo C Driver
-    - pushd mongo-c-driver; ./autogen.sh --enable-tests=no --enable-examples=no --with-libbson=bundled; make; sudo make install; popd;
+    - pushd mongo-c-driver
+
+    # TODO: Update this to our real minimum for the C++11 driver 3.1 release, once known.
+    - git checkout master
+
+    - ./autogen.sh --enable-tests=no --enable-examples=no --with-libbson=bundled; make; sudo make install
```

Comment:
I think this should match the autogen line in .mci.yml, right?  At least, it should have the `--disable-automatic-init-and-cleanup` flag.